packages/dinero.js/tsconfig.json

```diff
@@ -12,4 +11,5 @@
     { "path": "../currencies" }
   ],
   "include": ["src"],
-  "exclude": [ "**/__tests__/**/*"]}
```

Comment:
This formatting mistake in all the `tsconfig.json` files made it through my previous PR, sorry